### DISCUSSION TITLE
_site folder generation - when?

### DISCUSSION THREAD

**Discussion by brfignoni** [11ty/eleventy#2942] [Q&A]
Just a simple question.
Should I upload the _site folder generated on my local environment to production site, or should that happen on a build process on github, then automatic deploy to production?

**Comment by pdehaan** [comment]
Probably just depends on your preferences and complexity.

A lot of the sites I've built are just very basic and maybe only something that gets generated very infrequently so I'll just build+test locally and then deploy to Netlify/Vercel/GCP manually (while still keeping all the source code in GitHub).

If it's a bigger project, or has multiple people, I'd probably say offload the build+deploying to GitHub Actions/Netlify whenever you push a commit or make a tag. You could even set up a cron task in GitHub Actions to automatically deploy your site _n_-times per day/week on a schedule.
